CardChoiceSpawnUniqueCardPatch/CardChoiceSpawnUniqueCardPatch.cs

```diff
@@ -17,13 +17,14 @@ namespace CardChoiceSpawnUniqueCardPatch
 {
     [BepInDependency("com.willis.rounds.unbound", BepInDependency.DependencyFlags.HardDependency)]
     [BepInDependency("pykess.rounds.plugins.moddingutils", BepInDependency.DependencyFlags.HardDependency)]
-    [BepInPlugin(ModId, ModName, "0.1.8")]
+    [BepInPlugin(ModId, ModName, "0.1.10")]
```

Comment:
Why did the version rise by two?

Reply:
Because this incorporates another live PR that already increased the version number to 0.1.9.